converter/fluka/cards/beam_card.py

```diff
@@ -0,0 +1,72 @@
+from dataclasses import dataclass, field
+from converter.common import format_float
+from converter.fluka.cards.card import Card
+from converter.fluka.helper_parsers.beam_parser import BeamShape, FlukaBeam
+
+
+@dataclass
+class BeamCard:
+    """Class representing description of beam in Fluka input"""
+
+    data: FlukaBeam = field(default_factory=lambda: FlukaBeam())  # skipcq: PYL-W0108
+
+    def __str__(self) -> str:
+        """Return the card as a string."""
+        beam_card = Card(tag="BEAM")
+        if self.data.shape == BeamShape.GAUSSIAN:
+            shape_what = -1
+            x_y_multiplier = -1
+        elif self.data.shape == BeamShape.CIRCULAR:
+            shape_what = -1
+            x_y_multiplier = 1
+        else:
+            shape_what = 0
+            x_y_multiplier = 1
+        energy = format_float(self.data.energy*-1, 10)
```

Comment:
```suggestion
        energy = format_float(-self.data.energy, 10)
```

Reply:
Maybe better to rename this quantity to `momentum_or_energy`
The notion of handling something as negative number should very clean to user.
Add here a comment mentioning that `self.data.energy` is storing energy as positive number, explain in which units.
Then say what is stored in `momentum_or_energy`